homework9/src/components/Input/input.style.css

```diff
@@ -0,0 +1,86 @@
+.input-box {
+  width: 304px;
+}
+
+.input-box input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #c2d2d2;
+  border-radius: 10px;
+  outline-color: #1b6c71;
+  padding: 0 16px;
+  font-family: "Poppins-Light";
+}
+
+.input-box input::placeholder {
+  color: #93a3a3;
+}
+
+.input-apply {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 16px;
+  height: 40px;
+}
+
+.input-apply p {
+  align-self: center;
+  margin: 0;
+}
+
+.react-select__control {
+  width: 304px !important;
```

Comment:
can u avoid `!important`?

Reply:
I am manipulating an external library to suit the us design .. that is the only way to override it